templates/zerver/help/change-who-can-join-a-stream.md

```diff
@@ -0,0 +1,39 @@
+# Change who can join a stream
+Zulip allows you to create new private streams as a user with stream creation privileges.
```

Comment:
Yeah, I'll include a description of private streams. Also, I think any user in a private stream can invite another user to join the stream.